userguide/aws/how-tos/add-a-certificate.adoc

```diff
@@ -0,0 +1,157 @@
+
+The next page will give you the option to import a certificate, or to request a public or private certificate. Choose *private*, then click again on *Request a Certificate*. This will take you to the *Request a Certificate* page. You are initially at *Step 1: Add Domain Names*.
+
+image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-request-certificate.png[align=center]
+
+=== 2. Setup a CNAME
+
+To validate your certificate, you must first setup a *temporary* CNAME. The values to use for the CNAME will be seen shortly.  You will then create the CNAME that you will actually use to access the system.
+
+Enter your domain name, using the wildcard format. Then click *Next*. On the following page, select *DNS Validation*. The next page gives you the chance to assign one or more tags to your certificate. These are optional and can be skipped. Finally, you are given a chance to review your request. Check everything, then click *Confirm and Request*. The *Validation* page will appear.
+
+image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-cert-validation-1.png[align=center]
+
+At the bottom you will see your domain name with a status of *Pending Validation*. Click the triangle next to your domain name. The display will expand to show the values that you must use to create your validation CNAME.
+
+image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ELB-cert-validation-2.png[align=center]
+
+The two strings labeled `Name` and `Value` should be copied to the two required entries for your CNAME. Copy these long strings carefully. Do not include any trailing periods. There is an option to save these strings in a file so they may be copied more easily, or stored for a later time.
+
+Your second CNAME is much simpler. The `Name` entry will be `kaui`. The `Value` entry will be simply your domain name, e.g., `mydomain.com`.
+
+=== 3. Create and Validate your Certificate
+
+When your CNAMEs are set, return to ACM and click *Continue*. The display changes to show that your certificate is *Pending Validation*. The certificate should now appear also on the main ACM page. When (and if) it is validated, the status will change to *Success*. *This may take several hours or more.* A short time later it will change again to *Issued*.
+
+When your certificate is successfully created, you may delete the temporary (validation) CNAME from your DNS Profile.
+
+
+== Using Let's Encrypt (with Certbot)
+
+`Let's Encrypt` is a root CA that is free to use. Certificates based on `Let's Encrypt` can be created using the application `certbot`. This approach is useful when using a third-party load balancer like `nginx`, which cannot be protected with ACM certificates. This is the load balancer used with the single-tier Kill Bill option.`Let's Encrypt` can be used free of charge.
```

Comment:
Adding a link to Let's Encrypt (https://letsencrypt.org/ - if this is the correct link) might help.

Reply:
Done